src/solvers/dgsem_p4est/dg_3d_subcell_limiters.jl

```diff
@@ -0,0 +1,301 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+function create_cache(mesh::P4estMesh{3},
+                      equations, volume_integral::VolumeIntegralSubcellLimiting,
+                      dg::DG, uEltype)
+    cache = create_cache(mesh, equations,
+                         VolumeIntegralPureLGLFiniteVolume(volume_integral.volume_flux_fv),
+                         dg, uEltype)
+
+    A4dp1_x = Array{uEltype, 4}
```

Comment:
Do you know what the `dp1` stands for? We have it in multiple places 

https://github.com/search?q=repo%3Atrixi-framework%2FTrixi.jl+dp1+path%3A%2F%5Esrc%5C%2F%2F&type=code

but the abbreviation is not really clear to me, and I thus question its usefulness. 
My best guess would be double precision 1 (not sure what the 1 is for).

In my opinion, we could completely remove this `dp1` stuff (but then in a separate PR).

Reply:
I always thought that it is something like `dimension plus 1` since the array is larger in one direction. But I'm not sure.
If it doesn't make a performance difference, I strongly agree with removing this!

Reply:
Maybe we should ask that in the next Trixi meeting :sweat_smile:

Reply:
> I always thought that it is something like `dimension plus 1` since the array is larger in one direction.

Ah yeah that makes sense. Maybe we should add a comment at the relevant places and shorten the name still.

Reply:
The thing is: The dimensionality of the array is present anyways, so I do not see any benefit of having this